pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -41,6 +41,7 @@ type connTrackSystem struct {
 	connTrack            NetFilterConnTrack
 }
 
+// TODO: detect the endianness of the system when initializing conntrack dumper
```

Comment:
you should move the comment to https://github.com/antrea-io/antrea/blob/5c5cae5108ccaff0e1c93e9b99eae43b1b05aeab/pkg/agent/flowexporter/connections/connections.go#L177-L178 and provide more context. It's really out of place here.

Reply:
@srikartati I assumed the reason Srikar let me add a comment in the conntrack_linux.go is to keep the code parsing label as little Endian unchanged in connections.go?

Reply:
Yes, that is my intention. I think Antonin feels directly using the little-endian format lacks context.
Therefore, you could add a comment there in connections.go saying that we always expect labels from conntrack dumper to be added in little-endian format to make it clear. Please add the reason that the labels are stored in the same format in kernel datapath on most platforms.

Reply:
Thanks Srikar and Antonin, the comments have been updated.